stdlib/3/typing.pyi

```diff
@@ -339,14 +339,16 @@ class MutableSet(AbstractSet[_T], Generic[_T]):
     def __ixor__(self, s: AbstractSet[_S]) -> MutableSet[Union[_T, _S]]: ...
     def __isub__(self, s: AbstractSet[Any]) -> MutableSet[_T]: ...
 
-class MappingView:
+class MappingView(Sized, Iterable[_T]):
+    def __init__(self, mapping: Mapping[_KT_co, _VT_co]) -> None: ...  # undocumented
+    def __iter__(self) -> Iterator[_T]: ...
```

Comment:
This method doesn't actually exist at runtime on `collections.abc.MappingView`.

Reply:
I don't recall why I added that. Maybe I mistakenly assumed that `Sized` defined `__iter__()` and I wanted to make the type more specific. I'll remove the `__iter__()` changes from the patch and push an updated version.

Reply:
Correction pushed and passed CI.